Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -68,6 +69,14 @@ foreach ($suite in $suites)
     }    
 }
 
+$testMachinesGroupName=$testMachineGroup
+if ([string]::Equals($env:UseDtaV2Agent, "true", [System.StringComparison]::OrdinalIgnoreCase)) {
+    # dtav2://env/TestProject/_apis/testrig/<name>
+    $testMachinesGroupName=[string]::Format("dtav2://env/{0}/_apis/testrig/{1}", $env:SYSTEM_TEAMPROJECT, $testMachineGroup.ToLowerInvariant())
```

Comment:
talked with Shiva. Its cant be null. 

---
In reply to: [104144375](https://github.com/Microsoft/vsts-tasks/pull/3727#discussion_r104144375) [](ancestors = 104144375)